crates/re_types/src/archetypes/arrows3d.rs

```diff
@@ -14,11 +14,11 @@
 #![allow(clippy::too_many_lines)]
 #![allow(clippy::unnecessary_cast)]
 
-/// **Archetype**:  A batch of 3D arrows with optional colors, radii, labels, etc.
```

Comment:
I just noticed now that there is an extra space in there

Reply:
fixing in a tiny follow-up